recipes/intel-opencl-runtime/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -ex
+
+src="$SRC_DIR/$PKG_NAME"
+mkdir -p $PREFIX/lib/intel-ocl-cpu
+
+# Move to intel-ocl-cpu to avoid clashes with intel-opencl-clang
+cp -rv "$src/lib/"* "$PREFIX/lib/intel-ocl-cpu/"
```

Comment:
@tomashek, we are moving the package to a different root so that `libcommon_clang.so` in the `opencl_rt` package doesn't conflict with out `intel-opencl-clang` package.